src/modules/sort-pets.js

```diff
@@ -0,0 +1,24 @@
+import renderPets from './render-pets';
+
+function sortPetsByName(e, pets, petsContent) {
+  if (!pets) return;
+  if (!petsContent) return;
+  if (e.target.options.selectedIndex === 1) {
+    const sortPets = [...pets].sort((a, b) => {
+      const aName = a.name;
+      const bName = b.name;
+      return aName >= bName ? 1 : -1;
+    });
+    renderPets(sortPets, petsContent);
+  }
+  if (e.target.options.selectedIndex === 2) {
```

Comment:
maybe just else {} here? 
or if we have in future a lot of sort options I would suggest to use `switch case`, the code will look cleaner
let's do here in train purpose